### PR TITLE
Add sockets

### PR DESCRIPTION
**Pull request by Mar1nka**

### CODE REVIEW

File: src/server/config/environment/development.js [hunk 1]
```diff
@@ -5,4 +5,5 @@ module.exports = {
   apiUrl: 'http://localhost:3001',
   apiInternalUrl: 'http://api:3001',
   landingUrl: 'http://localhost:3000',
+  websocketsUrl: 'http://localhost:8082',
```
Comment (SobakaSlava) [inline src/server/config/environment/development.js]: I think it should be `webSocketUrl` (camelCase and in the singular)

File: src/server/config/routes.js [hunk 2]
```diff
@@ -24,6 +24,7 @@ indexRouter.get(/^((?!\.).)*$/, async (ctx) => {
     isDev: config.isDev,
     config: {
       apiUrl: config.apiUrl,
+      websocketsUrl: config.websocketsUrl,
```
Comment (SobakaSlava) [inline src/server/config/routes.js]: The same here

File: src/server/config/environment/development.js (hunk 1 above)
Comment (Mar1nka) [reply src/server/config/environment/development.js]: Ok

File: src/client/services/socketIo.service.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import io from 'socket.io-client';
+import { store } from 'resources/store';
+import { UPDATE_USER } from 'resources/user/user.actions';
+
+const socket = io(window.config.webSocketUrl, { transports: ['websocket'], query: { token: window.token } });
+
+socket.on('connect', () => {
+  console.log('WS connected'); //eslint-disable-line
+  socket.emit('subscribe', window.user._id);
```
Comment (IharKrasnik) [inline src/client/services/socketIo.service.js]: Subscribe to `user-${user._id}` from client side, instead than just `user._id`
(
check corresponding comment from API PR
https://github.com/paralect/koa-api-starter/pull/33
)

Comment (IharKrasnik) [inline src/client/services/socketIo.service.js]: Subscribe to `user-${user._id}` room from client side instead than just `user._id`

Check corresponding comment from API starter PR:
paralect/koa-api-starter#33

File: src/client/services/socketIo.service.js [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+  socket.on('user:updated', (user) => {
```
Comment (IharKrasnik) [inline src/client/services/socketIo.service.js]: The handler logic should be placed outside `socketIo.service.js` itself. 

`socketIo.service` should expose `on`, `emit`, `subscribe` and `unsubscribe` methods that should be used in handler files

**Example usage:**

```
import socketService from 'services/socketIo.service.js';

socketService.on('user:updated', user => store.dispatch(...));
```

**Another use-case for the future usage with subscribe call**

```
componentWillMount() {
   socketService.subscribe(`chat-1234`);
   socketService.on('message:created', ...);
}

componentWillUnmount() {
  socketService.unsubscribe(`chat-1234`);
}
```

**Pull request merged**